source/_integrations/energyid.markdown

```diff
@@ -0,0 +1,161 @@
+{% endconfiguration_basic %}
+
+## Key features
+
+### Data handling
+
+- **Smart Upload Throttling**: Minimum 5-minute interval between updates
+- **Value Validation**: Automatic filtering of non-numeric states
+- **Connection Resilience**:
+  - 3 retry attempts with exponential backoff
+  - Automatic reconnection after network failures
+  - Configurable retry intervals (1s, 2s, 4s)
+
+### Metric types
+
+- **Cumulative**: For ever-increasing counters (e.g., total kWh)
+- **Total**: Interval-based consumption (e.g., daily usage)
+- **Delta**: Change between measurements
+- **Gauge**: Instantaneous readings (e.g., temperature)
+
+### Data resolution options
+
+| Interval | Description | Subscription required |
+|----------|-------------|-----------------------|
+| PT5M     | 5-minute    | Premium               |
+| PT15M    | 15-minute   | Premium               |
+| PT1H     | Hourly      | Free                  |
+| P1D      | Daily       | Free                  |
+
+## Installation guide
+
+### Obtaining a webhook URL
+
+1. Log in to your EnergyID account
+2. Navigate to **Integrations** → **Incoming Webhooks**
+3. Create new webhook with:
+   - Target record
+   - Descriptive name
+4. Copy the generated URL (format: `https://app.energyid.eu/integrations/WebhookIn/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx`)
+
+### Home Assistant setup
+
+1. {% my config_flow_start title="Add EnergyID Integration" %} via Home Assistant UI
+2. Paste Webhook URL when prompted
+3. Select entity from dropdown list
+4. Choose matching EnergyID metric type
+5. Configure advanced options if needed
+
+## Data flow management
+
+### Update behavior
+
+- Immediate upload on state change
+- Minimum 5-minute interval between updates (configurable)
+- Last valid value persists during connection issues
+
+### Connection monitoring
+
+- Automatic health checks every 15 minutes
+- System log entries for connection status changes
+- Visual indicator in Home Assistant UI
+
+## Known limitations
+
+- **One-way synchronization**: Data flows only from Home Assistant → EnergyID
+- **Premium features**: Sub-1hour intervals require EnergyID subscription
+- **Entity limitations**: Each sensor requires separate webhook configuration
+
+## Troubleshooting
+
+### Common issues
+
+- **`Invalid webhook URL`**: Regenerate in EnergyID and update configuration.
+- **`Non-numeric state`**: Check sensor's unit_of_measurement.
+- **`Connection lost`**:
+  1. Verify network connectivity.
+  2. Check EnergyID service status.
+  3. Validate webhook URL is still active.
+
+### Diagnostic steps
+
+For detailed troubleshooting guidance, refer to Home Assistant's [troubleshooting documentation](/docs/configuration/troubleshooting/#enabling-debug-logging).
+
+## Removing the integration
+
+1. Go to {% my integrations title="**Settings** > **Devices & Services**" %}
+2. Locate EnergyID integration card
+3. Select **Configure**
+4. Choose **Delete Integration**
```

Comment:
```suggestion
```
These steps are redundant. They are part of the include.

![image](https://github.com/user-attachments/assets/a8e44ddb-e52c-4a8c-b75e-dc44588a7276)